src/main/kotlin/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,29 @@
+package calculator
+
+private const val EXPRESSION_DELIMITER = " "
+private const val EVEN_AND_ODD_STANDARD = 2
+private const val NUMBER_POSITION = 0
+
+class StringCalculator(
+    private val expression: String,
+) {
+    init {
+        require(!expression.isNullOrBlank())
+    }
+
+    fun calculate(): Int {
+        val numbersAndOperators = expression.split(EXPRESSION_DELIMITER)
```

Comment:
코틀린은 자바스크립트처럼 구조분해 기능을 지원하고 있습니다~!
단, [`data` 클래스](https://kotlinlang.org/docs/data-classes.html#data-classes-and-destructuring-declarations)에 한해서 입니다!
이를 이용하면 아래에서 사용하신 것처럼 first, second 처럼 모호한 프로퍼티를 호출하는 것 보다 보다 나은 사용을 할 수 있을것 같아요~!
```suggestion
        val (numbers, operators) = expression.split(EXPRESSION_DELIMITER)
```